devices/mecrator.js

```diff
@@ -79,4 +79,44 @@ module.exports = [
             await reporting.fanMode(endpoint);
         },
     },
+    {
+        fingerprint: [{modelID: 'TS011F', manufacturerName: '_TZ3210_pfbzs1an'}],
+        zigbeeModel: ['TS011F'],
+        model: 'SPUSB02',
+        vendor: 'Mercator',
+        description: 'Ikuü Double Power Point with USB',
+        fromZigbee: [fz.on_off, fz.electrical_measurement, fz.metering, fz.ignore_basic_report, fz.tuya_switch_power_outage_memory],
+        toZigbee: [tz.on_off],
+        exposes: [
+            e.switch().withEndpoint('left'),
+            e.switch().withEndpoint('right'),
+            e.power().withEndpoint('left'),
+            e.current().withEndpoint('left'),
+            e.voltage().withEndpoint('left').withAccess(ea.STATE),
+            e.energy(),
+        ],
+        endpoint: (device) => {
+            return {left: 1, right: 2};
+        },
+        // The configure method below is needed to make the device reports on/off state changes
+        // when the device is controlled manually through the button on it.
+        meta: {multiEndpoint: true},
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint1 = device.getEndpoint(1);
+            const endpoint2 = device.getEndpoint(2);
+            await reporting.bind(endpoint1, coordinatorEndpoint, ['genBasic', 'genOnOff', 'haElectricalMeasurement', 'seMetering']);
+            await reporting.bind(endpoint2, coordinatorEndpoint, ['genOnOff']);
+            await reporting.onOff(endpoint1);
+            await reporting.onOff(endpoint1);
+            await reporting.onOff(endpoint1);
+            await reporting.rmsVoltage(endpoint1, {change: 5});
+            await reporting.rmsCurrent(endpoint1, {change: 50});
+            await reporting.activePower(endpoint1, {change: 1});
+            await reporting.onOff(endpoint1);
```

Comment:
Why is ` await reporting.onOff(endpoint1);` done so many times? 1 time should be enough.

Reply:
> Why is ` await reporting.onOff(endpoint1);` done so many times? 1 time should be enough.

Hey @Koenkk,

Thanks so much for reviewing the pull request right away.

The short of it is that I don't really know and I was wondering the same thing.

The original code comes from https://community.home-assistant.io/t/mercator-ikuu-double-outdoor-powerpoint/343779 by [jasonc](https://community.home-assistant.io/u/jasonc) and then it was adapted by [dortamur](https://github.com/dortamur) as per [this post](https://github.com/Koenkk/zigbee2mqtt/issues/11272#issuecomment-1068064625)

I can confirm, however, after testing it for about a week, it's working really well. I would be happy to adjust the `awaits` as you see fit and confirm if the change has any side effects.

Let me know next steps.

Cheers, Alex

Reply:
Please check if it still reports its on/off state after removing the duplicate `await reporting.onOff(endpoint1);`, restart z2m; repair device

Reply:
Thank you for getting back,
We have tested for over a week with no issues after removing duplicate `await` lines.